Source/Core/DolphinQt/CheatsManager.cpp

```diff
@@ -175,11 +175,12 @@ void CheatsManager::OnStateChanged(Core::State state)
   if (state != Core::State::Running && state != Core::State::Paused)
     return;
 
-  auto* model = Settings::Instance().GetGameListModel();
+  if (m_game_list_model)
```

Comment:
I've fixed it here in `CheatsManager.cpp`, and in `GameListDialog.cpp`.

I've now confirmed that the **Cheats Manager** dialog correctly displays tabs, and the codes if present. Also, I have verified that the **Select a game** dialog, used for switching games after the NetPlay session has been listed, also displays the list of games now.

Thanks for catching this; it would have been even more embarrassing.